src/main/java/games/strategy/engine/lobby/server/IModeratorController.java

```diff
@@ -40,46 +30,19 @@ default void banUsername(INode node, Date banExpires) {
   default void banIp(INode node, Date banExpires) {}
 
   /**
-   * @deprecated Kept to maintain backwards compatibility.
-   *             Remove with next incompatible release.
-   */
-  @Deprecated
-  default void banMac(INode node, Date banExpires) {
-    zzBanMac(node, banExpires != null ? banExpires.toInstant() : null);
-  }
-
-  /**
-   * @deprecated Kept to maintain backwards compatibility.
-   *             Remove with next incompatible release.
-   */
-  @Deprecated
-  default void banMac(final INode node, final String hashedMac, final Date banExpires) {
-    zzBanMac(node, banExpires != null ? banExpires.toInstant() : null);
```

Comment:
There was a copy-paste error here that I fixed when moving this method to `ModeratorController`.  This method is delegating to the two-parameter version of `zzBanMac`, and thus is ignoring the `hashedMac` parameter; it should have delegated to the three-parameter version.